ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -120,9 +145,9 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
   /**
    * Determine if we show host modal or quick connect based on target attributes.
    * @param {TargetModel} model
+   * @param {function} toggleModal
    */
   @action
-  @loading
   async preConnect(model, toggleModal) {
```

Comment:
Do we not need this `@loading` here anymore?

Reply:
No. The main `connect` has `@loading` on it so it handles that for all instances of "connect". I tested this to make sure.